diff --git a/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java b/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java
index xxxxxxx..xxxxxxx 100644
--- a/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java
+++ b/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java
@@ -1030,7 +1030,7 @@ private static void setBigPictureStyle(
 }
 
 if (bigPictureStyleInformation.hideExpandedLargeIcon) {
-  bigPictureStyle.bigLargeIcon(null);
+  bigPictureStyle.bigLargeIcon((Bitmap) null);
 } else {
   if (bigPictureStyleInformation.largeIcon != null) {
     bigPictureStyle.bigLargeIcon(
